warehouse/oso_sqlmesh/models/intermediate/events/int_events__github.sql

```diff
@@ -7,204 +7,149 @@ MODEL (
   ),
   start @github_incremental_start,
   cron '@daily',
+  dialect trino,
   partitioned_by (DAY("time"), "event_type"),
   grain (time, event_type, event_source, from_artifact_id, to_artifact_id)
```

Comment:
```suggestion
  grain (time, event_type, event_source, from_artifact_id, to_artifact_id),
  audits (
    not_null(columns := (to_artifact_name, to_artifact_namespace, event_type, event_source_id, event_source))
  )
```
This will ensure that we don't have nulls in these columns